packages/experimental/Menu/src/MenuTrigger/useMenuTrigger.ts

```diff
@@ -1,15 +1,33 @@
 import { useMenuContext } from '../context/menuContext';
 import { InteractionEvent } from '@fluentui-react-native/interactive-hooks';
-import { MenuTriggerProps } from './MenuTrigger.types';
-import { Platform } from 'react-native';
+import { MenuTriggerProps, MenuTriggerState } from './MenuTrigger.types';
+import { AccessibilityActionEvent, AccessibilityActionName, Platform } from 'react-native';
+import React from 'react';
 
-export const useMenuTrigger = (_props: MenuTriggerProps) => {
+const accessibilityActions = [{ name: 'Expand' as AccessibilityActionName }, { name: 'Collapse' as AccessibilityActionName }];
```

Comment:
Mostly a note to self, this differs on macOS, where I think the action is "showMenu". I didn't implement that for ContextualMenu, so I'll need to look into that for the V1 macOS control. Just an FYI.

Reply:
Did some platform gating so that places you need to fix/consider are more obvious.